src/sections/safe.tsx

```diff
@@ -1,28 +1,123 @@
 import { useEffect } from 'react';
-import { Store } from '../types/index';
 
 //Stores
 import { useAppDispatch, useAppSelector } from '../store';
+import { actionsAsync } from '../store/slices/safe/actionsAsync';
 
 // HOPR Components
 import Section from '../future-hopr-lib-components/Section';
 import { useSigner } from '../hooks';
-import { actionsAsync } from '../store/slices/safe/actionsAsync';
-
+import { utils } from 'ethers';
 function SafeSection() {
   const dispatch = useAppDispatch();
-  const safe = useAppSelector((store: Store) => store.safe);
+  const safe = useAppSelector((store) => store.safe);
   const { signer } = useSigner();
 
   useEffect(() => {
     if (signer) {
       dispatch(actionsAsync.getSafesByOwnerThunk({ signer }));
     }
   }, [signer]);
+
   return (
     <Section className="Section--safe" id="Section--safe" yellow>
-      SAFE REDUX STORE
+      <h1>Safe redux store</h1>
       <pre>{JSON.stringify(safe, null, 4)}</pre>
```

Comment:
Please switch around the store and actions, so `<pre>{JSON.stringify(safe, null, 4)}</pre>` is below actions